### PR TITLE
WIP - Target Rider 2021.3 EAP 5

### PR DESCRIPTION
**Pull request by maartenba**
This PR:
* Updates Kotlin to 1.5.31
* Removed the stdlib from the plugin
* Fixes an issue with publishing
* Fixes https://youtrack.jetbrains.com/issue/RIDER-69312 and https://youtrack.jetbrains.com/issue/DEXP-626150

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties [hunk 1]
```diff
@@ -4,29 +4,30 @@ javaVersion=11
 org.gradle.jvmargs='-Duser.language=en'
 sources=false
 
-intellij_version=IC-213.3714-EAP-CANDIDATE-SNAPSHOT
-rider_version=RD-2021.3-EAP4-SNAPSHOT
+intellij_version=IC-213.4928-EAP-CANDIDATE-SNAPSHOT
+rider_version=RD-2021.3-SNAPSHOT
```
Comment (sdubov) [inline PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties]: Do we skip `EAP5` in version?

File: PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/component/appservice/AppExistingComponent.kt [hunk 2]
```diff
@@ -106,7 +107,7 @@ abstract class AppExistingComponent<TApp : WebAppBase, TSlot: DeploymentSlotBase
 
         val osColumnIndex = (pnlExistingAppTable.table.model as DefaultTableModel).findColumn(pnlExistingAppTable.appTableColumnOs)
         require(osColumnIndex >= 0) { "Column index is out of range" }
-        sorter.rowFilter = regexFilter(OperatingSystem.WINDOWS.name.toLowerCase().capitalize(), osColumnIndex)
+        sorter.rowFilter = regexFilter(OperatingSystem.WINDOWS.name.lowercase().replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }, osColumnIndex)
```
Comment (sdubov) [inline PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/component/appservice/AppExistingComponent.kt:110]: Is that for Linux?

File: PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties (hunk 1 above)
Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties]: Not released yet :-) Will update when we have it.

File: PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/component/appservice/AppExistingComponent.kt (hunk 2 above)
Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/component/appservice/AppExistingComponent.kt:110]: Suggested after updating Kotlin version, old method was deprecated. Apparently `capitalize()` is going away

**Review by sdubov — APPROVED** [review]

**Pull request merged**